dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/CustomFormParam.tsx

```diff
@@ -14,7 +14,14 @@ export default function CustomFormComponentLoader({
   handleBasicFormParamChange,
 }: ICustomParamProps) {
   // Fetches the custom-component bundle served by the dashboard nginx
-  const url = `${window.location.origin}/custom_components.js`;
+
+  const {
+    config: { remoteComponentsUrl },
+  } = useSelector((state: IStoreState) => state);
+
+  const url = remoteComponentsUrl
+    ? remoteComponentsUrl
+    : `${window.location.origin}/custom_components.js`;
```

Comment:
If the url is unreachable, which error is thrown to the user? I'm guessing it's an internal 404 that simply results in this component not being loaded, but I want to double-check it.

Reply:
Yes correct there is a 404 and `param` object that specified a custom component will not load and will generate an section with `Error` in plain text